configure.ac

```diff
@@ -20,8 +20,8 @@ AC_INIT([netCDF],[4.9.4-development],[support-netcdf@unidata.ucar.edu],[netcdf-c
 : ${CFLAGS=""}
 
 AC_SUBST([netCDF_VERSION_MAJOR]) netCDF_VERSION_MAJOR=4
-AC_SUBST([netCDF_VERSION_MINOR]) netCDF_VERSION_MINOR=9
-AC_SUBST([netCDF_VERSION_PATCH]) netCDF_VERSION_PATCH=4
+AC_SUBST([netCDF_VERSION_MINOR]) netCDF_VERSION_MINOR=10
+AC_SUBST([netCDF_VERSION_PATCH]) netCDF_VERSION_PATCH=0
```

Comment:
Do I understand correctly this broke the ABI of the library between v4.9.2 and v4.9.3 (since this change is in v4.9.3) because the soversion changed from 19 to 22?

Reply:
The current release tarball declares the version as `4.9.3`, as expected. Maybe this was an inadvertent change that was reverted?

Reply:
But here:

https://github.com/Unidata/netcdf-c/blob/3a6b1debf1557f07b606ce3653e44f0d711203be/configure.ac#L28-L32

Reply:
A question to the NetCDF developers: Was the SO_VERSION change from 4.9.2 to 4.9.3 intentional or accidental? Is the new ABI incompatible to 4.9.2?

Reply:
If I understand the changelog correctly:
- a lot of configuration and testing changes
- some bug fixes
- one API/ABI change https://github.com/Unidata/netcdf-c/pull/2743/files
So, technically indeed a breaking change.